### PR TITLE
selectors are grouped close to Atomic model (whatever it means)

### PR DESCRIPTION
**Pull request by aleksandrAndrosov**

### CODE REVIEW

File: src/scss/molecules/_header.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    position: relative;
+    right: 12px;
+    top: 2px;
+    height: 25px;
+    width: 42px;
+    background-color: white;
+    border: 3px orange solid;
+    border-radius: 20px;
+  }
+  nav {
```
Comment (dnesterenko27) [inline src/scss/molecules/_header.scss]: empty - remove

File: src/scss/molecules/_providers.scss [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+  .icons {
+    position: relative;
+    @extend .fx-c-c;
+    flex-wrap: wrap;
+
+    .wrapper {
+      @extend .fx-c-c;
+      position: relative;
+      width: 42%;
```
Comment (dnesterenko27) [inline src/scss/molecules/_providers.scss]: why 42%?

File: src/scss/molecules/_providers.scss [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+      overflow: hidden;
+    }
+
+    .wrapper:hover:before {
+      @extend .ps-empty-abs;
+      display: block;
+
+      width: 150%;
+      height: 150%;
+      background: linear-gradient(#ebff0f 0%,#206018 24%,#f23cbb 53%,#1b1268 80%,#dbf043 100%);
```
Comment (dnesterenko27) [inline src/scss/molecules/_providers.scss]: (#ebff0f 0%, #206018 24%, #f23cbb 53%, #1b1268 80%, #dbf043 100%) add spaces before #

File: src/scss/molecules/_welcome-registration.scss [hunk 4]
```diff
@@ -0,0 +1,26 @@
+// ==========================================================================
+// $Welcome-registration
+// Inlined to avoid required HTTP connection: h5bp.com/r
+// ==========================================================================
+.welcome-registration {
+  color: $welcome-registration-text;
+
+  * {
```
Comment (dnesterenko27) [inline src/scss/molecules/_welcome-registration.scss]: don't use *

File: src/scss/utilities/_colors.scss [hunk 5]
```diff
@@ -5,6 +5,10 @@
 // =============
 
 $color-button-registration: #8019cc;
+$color-button-registration-text: white;
```
Comment (dnesterenko27) [inline src/scss/utilities/_colors.scss]: don't use literals for colors

File: src/index.html [hunk 6]
```diff
@@ -48,7 +48,7 @@
 
 </head>
 <body>
-    <header>
+    <header class="fx-c-sb">
```
[the remark this answers is not in the document]
Comment (aleksandrAndrosov) [reply src/index.html]: display: flex, align-items: center, justify-content:space-between
it's a litle bit like atomic CSS

File: src/index.html [hunk 7]
```diff
@@ -70,78 +70,75 @@
...
     </div>
     <div class="accordion">
-        <ul class="accordion-list">
-            <li class="accordion-element">
-                <input class="accordion-check" type="checkbox" checked>
-                <span class="accordion-element-header">Languages Used</span>
-                <p class="accordion-element-text">This page was written in HTML and CSS. The CSS was compiled from SASS. I used Normalize as my CSS reset and -prefix-free to save myself some headaches. I haven't quite gotten the hang of Slim for compiling into HTML, but someday I'll use it since its syntax compliments that of SASS. Regardless, this could all be done in plain HTML and CSS.</p>
+        <ul>
+            <li>
+                <input type="checkbox" checked>
```
[the remark this answers is not in the document]
Comment (aleksandrAndrosov) [reply src/index.html]: even if it is not used?

File: src/index.html [hunk 8]
```diff
@@ -70,78 +70,75 @@
...
-            <img src="images/icons/8.jpg" class="icon">
-        </div>
-        <div class="icon-wrapper">
-            <div class="icon-wrapper-text">текст9</div>
-            <img src="images/icons/9.jpg" class="icon">
-        </div>
-        <div class="icon-wrapper">
-            <div class="icon-wrapper-text">...</div>
-            <img src="images/icons/10.jpg" class="icon">
+        <h1>Провайдеры</h1>
```
[the remark this answers is not in the document]
Comment (aleksandrAndrosov) [reply src/index.html]: should i use h2?

File: src/index.html [hunk 9]
```diff
@@ -70,78 +70,75 @@
...
+        <div class="icons">
+            <div class="wrapper">
+                <div class="text">у пиявок 32 мозга</div>
+                <img src="images/icons/1.jpg">
```
[the remark this answers is not in the document]
Comment (aleksandrAndrosov) [reply src/index.html]: ok

File: src/scss/molecules/_header.scss (hunk 1 above)
Comment (aleksandrAndrosov) [reply src/scss/molecules/_header.scss]: it's not in use for this request, sorry

File: src/index.html (hunk 6 above)
[the remark this answers is not in the document]
Comment (dnesterenko27) [reply src/index.html]: a bit complicated

File: src/index.html (hunk 7 above)
[the remark this answers is not in the document]
Comment (dnesterenko27) [reply src/index.html]: yep

File: src/scss/molecules/_providers.scss (hunk 3 above)
Comment (aleksandrAndrosov) [reply src/scss/molecules/_providers.scss]: ok

File: src/scss/molecules/_welcome-registration.scss (hunk 4 above)
Comment (aleksandrAndrosov) [reply src/scss/molecules/_welcome-registration.scss]: ok

File: src/scss/utilities/_colors.scss (hunk 5 above)
Comment (aleksandrAndrosov) [reply src/scss/utilities/_colors.scss]: ok

File: src/index.html (hunk 8 above)
[the remark this answers is not in the document]
Comment (dnesterenko27) [reply src/index.html]: any of *h2* to *h6*

File: src/index.html (hunk 6 above)
[the remark this answers is not in the document]
Comment (aleksandrAndrosov) [reply src/index.html]: i found something similar here https://acss.io/

[the remark this answers is not in the document]
Comment (aleksandrAndrosov) [reply src/index.html]: if in saas you extend a class like this with other classes, yo get less code in resulting css
.one, .two, .... , .etc {
style
}

[the remark this answers is not in the document]
Comment (aleksandrAndrosov) [reply src/index.html]: but i'm not sure, if it is good to use it

File: src/index.html (hunk 8 above)
[the remark this answers is not in the document]
Comment (aleksandrAndrosov) [reply src/index.html]: fixed

File: src/scss/atoms/_flex.scss [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+.fx-c-sb {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.fx-div {
+  position: relative;
+  margin: 2%;
```
[the remark this answers is not in the document]
Comment (aleksandrAndrosov) [reply src/scss/atoms/_flex.scss]: made a constant in constants
fixed
but need help in understanding what should be margin value. and why

File: src/scss/atoms/_pseudo.scss [hunk 11]
```diff
@@ -0,0 +1,4 @@
+.ps-empty-abs {
```
[the remark this answers is not in the document]
Comment (aleksandrAndrosov) [reply src/scss/atoms/_pseudo.scss]: for pseudo elements

File: src/scss/atoms/_texts.scss [hunk 12]
```diff
@@ -0,0 +1,9 @@
+.tx-upper {
+  text-transform: uppercase;
+}
+
+.tx-section-h1 {
+  font-size: 1.9em;
```
[the remark this answers is not in the document]
Comment (aleksandrAndrosov) [reply src/scss/atoms/_texts.scss]: fixed

File: src/scss/layout/_accordion.scss [hunk 13]
```diff
@@ -0,0 +1,9 @@
+// ==========================================================================
+// $Header
+// Inlined to avoid required HTTP connection: h5bp.com/r
+// ==========================================================================
+.accordion{
```
[the remark this answers is not in the document]
Comment (aleksandrAndrosov) [reply src/scss/layout/_accordion.scss]: fixed

File: src/scss/layout/_accordion.scss [hunk 14]
```diff
@@ -0,0 +1,9 @@
...
+  @extend .fx-c-c;
+  position: relative;
+  margin: 2%;
```
[the remark this answers is not in the document]
Comment (aleksandrAndrosov) [reply src/scss/layout/_accordion.scss]: fixed

File: src/scss/layout/_body.scss [hunk 15]
```diff
@@ -0,0 +1,19 @@
...
+// $Body
+// Inlined to avoid required HTTP connection: h5bp.com/r
+// ==========================================================================
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  background-image: url("../images/bg.jpg");
+  background-color: coral;
```
[the remark this answers is not in the document]
Comment (aleksandrAndrosov) [reply src/scss/layout/_body.scss]: fixed

File: src/scss/layout/_welcome-registration.scss [hunk 16]
```diff
@@ -0,0 +1,12 @@
+// ==========================================================================
+// $Welcome-registration
+// Inlined to avoid required HTTP connection: h5bp.com/r
+// ==========================================================================
+.welcome-registration {
+  position: relative;
+  //height: 153px;
```
[the remark this answers is not in the document]
Comment (aleksandrAndrosov) [reply src/scss/layout/_welcome-registration.scss]: fixed

File: src/scss/molecules/_accordion-list.scss [hunk 17]
```diff
@@ -0,0 +1,149 @@
...
+    border-left: solid 1px white;
+    animation: flipdown 0.5s ease both 2s;
+    @extend .ps-empty-abs;
+  }
+
+  ul {
+    display: flex;
+    flex-direction: column;
+    list-style: none;
+    perspective: 900;
```
[the remark this answers is not in the document]
Comment (aleksandrAndrosov) [reply src/scss/molecules/_accordion-list.scss]: fixed. dont know

File: src/scss/molecules/_accordion-list.scss [hunk 18]
```diff
@@ -0,0 +1,149 @@
...
+  }
+
+  li {
+
+    display: block;
+    position: relative;
+    padding: 0;
+    margin: 0;
+    width: 100%;
+    padding-bottom: 4px;
```
[the remark this answers is not in the document]
Comment (aleksandrAndrosov) [reply src/scss/molecules/_accordion-list.scss]: fixed

File: src/scss/molecules/_accordion-list.scss [hunk 19]
```diff
@@ -0,0 +1,149 @@
...
+    &:before {
+      content: counter(section);
+
+      position: absolute;
+      left: -55px;
+      top: 10px;
+      padding: 9px;
+      width: 15px;
+      height: 15px;
+      display: flex;
```
[the remark this answers is not in the document]
Comment (aleksandrAndrosov) [reply src/scss/molecules/_accordion-list.scss]: fixed

File: src/scss/molecules/_accordion-list.scss [hunk 20]
```diff
@@ -0,0 +1,149 @@
...
+      justify-content: center;
+      align-content: center;
+
+      border-radius: 50%;
+      background-color: white;
```
[the remark this answers is not in the document]
Comment (aleksandrAndrosov) [reply src/scss/molecules/_accordion-list.scss]: fixed

File: src/scss/molecules/_accordion-list.scss [hunk 21]
```diff
@@ -0,0 +1,149 @@
...
+    }
+
+    &:after {
+      @extend .ps-empty-abs;
+      display: block;
+      left: -61px;
+      top: 4px;
+      width: 43px;
```
[the remark this answers is not in the document]
Comment (aleksandrAndrosov) [reply src/scss/molecules/_accordion-list.scss]: need help with this

File: src/scss/molecules/_accordion-list.scss [hunk 22]
```diff
@@ -0,0 +1,149 @@
...
+      height: 43px;
+
+      border-radius: 50%;
+      border: solid 1px white;
+    }
+
+    $elements : 6 ;
```
[the remark this answers is not in the document]
Comment (aleksandrAndrosov) [reply src/scss/molecules/_accordion-list.scss]: fixed

**Pull request merged**